CHANGELOG.md

```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.6.2] - 2024-10-10
```

Comment:
The link reference at the bottom of the file needs to be added.

Reply:
missed that. Done in https://github.com/Automattic/Co-Authors-Plus/pull/1064/commits/3e3b40f719f09e459c0d7698416222edd73b7061 (also fixed last release link for consistency)